x/storage/keeper/abci.go

```diff
@@ -38,11 +39,22 @@ func EndBlocker(ctx sdk.Context, keeper Keeper) {
 	}
 
 	// delete buckets
-	_, err = keeper.DeleteDiscontinueBucketsUntil(ctx, blockTime, deletionMax-deleted)
-	if err != nil {
-		ctx.Logger().Error("should not happen, fail to delete buckets, err " + err.Error())
-		panic("should not happen")
+	doDeleteBucket := true
+	if ctx.BlockHeight() > 5946511 && ctx.ChainID() == "greenfield_5600-1" {
```

Comment:
We can use "upgrade/types.TestnetChainID" instead
<img width="603" alt="image" src="https://github.com/bnb-chain/greenfield/assets/25412254/7c16915c-6424-4b4e-a538-a3c26b423eb6">